capabilities_test.go

```diff
@@ -8,22 +8,41 @@ import (
 
 var _ = Describe("Capabilities", func() {
 	var capabilities Capabilities
+	proxyType := "manual"
```

Comment:
No reason to extract these if we can use `string` instead of `*string` for the `ProxyConfig` fields.